common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/rest/client/vrli/AbstractRestClientVrli.java

```diff
@@ -33,114 +33,120 @@
 import java.util.Properties;
 
 public abstract class AbstractRestClientVrli extends RestClient {
```

Comment:
Can you revert the tab -> whitespace changes, can't review it like this and it was not necessary